### PR TITLE
Make use_ambiguous_login_error() more tolerant to broken user code

### PR DESCRIPTION
**Pull request by sjinks**
## Description

One of the client sites added an incorrect implementation of the `login_errors` filter, which under certain circumstances returned `null` instead of a string. This PR makes `use_ambiguous_login_error()` (which is our `login_errors` hook) more tolerant to such cases.

### Plugin Updated: VIP Security

`use_ambiguous_login_error()` is now more tolerant to broken user code.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [x] This change works and has been tested locally (or has an appropriate fallback).
- [ ] This change works and has been tested on a Go sandbox.
- [ ] This change has relevant unit tests (if applicable).
- [ ] This change has relevant documentation additions / updates (if applicable).
- [x] I've created a changelog description that aligns with the provided examples.

### CODE REVIEW

File: security/login-error.php [hunk 1]
```diff
@@ -10,11 +10,11 @@
  * 
  * @since 1.1
  */
-function use_ambiguous_login_error( string $error ): string {
+function use_ambiguous_login_error( $error ): string {
 	global $errors;
 
 	if ( ! is_wp_error( $errors ) ) {
-		return $error;
+		return (string) $error;
```
Comment (sjinks) [inline security/login-error.php:17]: If we are passed something other than a string, limit possible damage by explicitly casting the argument to a string.

Comment (rebeccahum) [reply security/login-error.php:17]: @sjinks Is this something we want to expose though since it gets directly outputted?

**Review by rebeccahum — APPROVED** [review]

**Comment by sjinks** [comment]
r2059-stacks

**Comment by jesse-greathouse** [comment]
Release v20211207.1

**Pull request merged**